src/content/en/fundamentals/primers/service-workers/debugging-service-workers.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: shared/narrow
+published: false
+title: "Debugging service workers"
+description: "So far our service worker doesn't do very much. But it's enough that we can watch it working. That makes this a good time to talk about debugging."
+authors:
+  - josephmedley
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 5
+key-takeaways:
+  tldr:   
+  - "Debugging service workers requires a few different tools including two different DevTools windows. You'll need one of those windows to debug install and activate." 
+---
+
+<p class="intro">
+  So far our service worker doesn't do very much. But it's enough that we can 
+  watch it working. That makes this a good time to talk about debugging.
+</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+## The `sw-primer` project
+
+The [`sw-primer` project](https://github.com/google/sw-primer), located at 
```

Comment:
I've sent you an invite to become a "Member". I'm not sure whether or not that will grant you the ability to create a new repo under GoogleChrome; let me know if you still can't.